ProQuest.js

```diff
@@ -499,6 +498,33 @@ function getItemType(types) {
 	return guessType;
 }
 
+function scrapeEbraryResults(doc, refs) {
+	// Since we can't chase URLs, let's get what we can from the page
+	var ebraryResults = ZU.xpath(doc, '//div[@class="ebraryitem"]');
```

Comment:
Build up this list in `getSearchResults` and pass a filtered list here based on user selection. E.g. https://github.com/zotero/translators/blob/e9423311b4a492dc7862c7d220f1ac5c23246b82/Cambridge%20Journals%20Online.js